src/py/mat3ra/made/tools/build/defect/__init__.py

```diff
@@ -13,15 +12,7 @@
     TerraceSlabDefectBuilder,
 )
 from .configuration import PointDefectConfiguration, AdatomSlabPointDefectConfiguration, IslandSlabDefectConfiguration
-from .enums import PointDefectTypeEnum
-
-
-class DefectBuilderFactory(BaseFactory):
-    __class_registry__ = {
-        PointDefectTypeEnum.VACANCY: "mat3ra.made.tools.build.defect.builders.VacancyPointDefectBuilder",
-        PointDefectTypeEnum.SUBSTITUTION: "mat3ra.made.tools.build.defect.builders.SubstitutionPointDefectBuilder",
-        PointDefectTypeEnum.INTERSTITIAL: "mat3ra.made.tools.build.defect.builders.InterstitialPointDefectBuilder",
-    }
+from .enums import PointDefectTypeEnum, DefectBuilderFactory
```

Comment:
Factory is not an enum, let's keep it in this file

Reply:
It's used in ./builders which is imported here, we'll have circular dependency, that's why enums were separated in the first place. The "factory" file will be only solution